tutorindigo/plugin.py

```diff
@@ -204,3 +165,49 @@ def _override_openedx_docker_image(
         ),
     ]
 )
+
+
+# Apply patches from tutor-indigo
+for path in glob(
+    os.path.join(
+        str(importlib_resources.files("tutorindigo") / "patches"),
+        "*",
+    )
+):
+    with open(path, encoding="utf-8") as patch_file:
+        hooks.Filters.ENV_PATCHES.add_item(
+            (os.path.basename(path), patch_file.read())
+        )
+
+
+for mfe in indigo_styled_mfes:
+    PLUGIN_SLOTS.add_items([
+        (
+            mfe,
+            "footer_slot",
+            """ 
+            {
+                op: PLUGIN_OPERATIONS.Hide,
+                widgetId: 'default_contents',
+            },
+            {
+                op: PLUGIN_OPERATIONS.Insert,
+                widget: {
+                    id: 'default_contents',
+                    type: DIRECT_PLUGIN,
+                    priority: 1,
+                    RenderWidget: <IndigoFooter />,
+                },
+            },
+            {
+            op: PLUGIN_OPERATIONS.Insert,
```

Comment:
```suggestion
                op: PLUGIN_OPERATIONS.Insert,
```

Some additional whitespace to make it line up. :)